tracer/test/Datadog.Trace.Tests/Logging/DirectSubmission/DirectSubmissionLogLevelExtensionsTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// <copyright file="DirectSubmissionLogLevelExtensionsTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq;
+using Datadog.Trace.Logging.DirectSubmission;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Logging.DirectSubmission
+{
+    public class DirectSubmissionLogLevelExtensionsTests
+    {
+        [Fact]
+        public void GetName_IsValidForAllLevels()
+        {
+            var allValues = Enum.GetValues(typeof(DirectSubmissionLogLevel))
+                                .Cast<DirectSubmissionLogLevel>();
+
+            foreach (var value in allValues)
+            {
+                value.GetName().Should().NotBeNullOrEmpty();
+            }
+        }
+
+        [Fact]
+        public void GetName_ThrowsForUnknownLogLevel()
+        {
+            Assert.Throws<InvalidOperationException>(() => ((DirectSubmissionLogLevel)123).GetName());
```

Comment:
Should we throw? Or should we default to something? Seeing as we're sniffing out logs, I'd hate to see this crash an app because of some custom log levels somewhere.

Reply:
I'd argue that as this is an enum _we_ define (not one coming from a library etc), then it requires egregious programming on our part that should be punished. 

(Yeah, fair point, better to log at the wrong level than kill anything)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1945/commits/7f655a451f63c85b0c2f961e65dd04892e996920